scripts/pusher-def.patch

```diff
@@ -1,20 +1,7 @@
+# move pusher stuff to own def
```

Comment:
Any explanation why?

Reply:
So you can just delete it?

Reply:
I guess it's just easier to plugin in the pusher_id stuff later https://github.com/famedly/dart_openapi_codegen/commit/1e048f4fa7cca5387842d76c8352cafa2e9de4fd

I would vote to just keep it because it makes more sense if you look at the pusher-id patch